can/simulator/rtos_main.cpp

```diff
@@ -81,7 +85,10 @@ static auto dispatcher_task =
  * The socket can reader. Reads from socket and writes to message buffer.
  */
 void can_bus_poll_task(void *) {
-    transport.open("vcan0");
+    const char *channel = std::getenv(ChannelEnvironmentVariableName);
+    channel = channel ? channel : DefaultChannel;
```

Comment:
Feels a little gross to be changing a `const char*` even though it's legal; maybe use a temporary?